### PR TITLE
BUG: unused param in het_white, docstring wrong

### PR DESCRIPTION
**Pull request by jbrockmendel**
This starts the process of, one-by-one, taking functions in sandbox.stats.diagnostic, ensuring they are tested, and moving them to stats.diagnostic.  non-sandbox should not be importing from sandbox.

Also sandbox.stats.diagnostic has a bunch of stuff labelled as "will be removed in 0.9" which can go as part of this sequence of PRs.

### CODE REVIEW

**Comment by jbrockmendel** [comment]
@ChadFulton this is one of a bunch of PRs working towards #4887.  It’ll take some teamwork to get @josef-pkt to push these through.

File: statsmodels/stats/diagnostic.py [hunk 1]
```diff
@@ -1,14 +1,103 @@
...
+from ._lilliefors import (  # noqa:F401
+    kstest_fit, lilliefors, lillifors, kstest_normal,
+    kstest_exponential)  # lillifors is deprecated
+from ._adnorm import normal_ad  # noqa:F401
+
+
+# -----------------------------------------------------------------
+# Misc Helpers
+
+class ResultsStore(object):
```
Comment (ChadFulton) [inline statsmodels/stats/diagnostic.py]: Where did this come from?

Comment (jbrockmendel) [reply statsmodels/stats/diagnostic.py]: Top of the sandbox file.  For now copied instead of moved.

**Comment by josef-pkt** [comment]
Not for merging
this pretty much destroys git blame
diagnostics can wait

**Comment by jbrockmendel** [comment]
> this pretty much destroys git blame

There are a number of places throughought the code where functions have comments to the effect of "moved from [...]", presumably for blame purposes.

Is there a method for moving a function to a more appropriate location that doesn't lead to this objection?  If not, it may be a "proves too much" situation.

In the meantime, bunches of really easy PRs, many of which could be delegated.

**Comment by bashtage** [comment]
> Is there a method for moving a function to a more appropriate location that doesn't lead to this objection? If not, it may be a "proves too much" situation.

You can copy a file out of sandbox, then restore the file to the sandbox to create an intentional conflict, and then you can have full git history for both. 

See 

https://stackoverflow.com/questions/16937359/git-copy-file-preserving-history/44036771#44036771

for the steps needed, or for a function, 

https://stackoverflow.com/a/53849613/2551705

This strategy is only of limited usefulness since you can't then move a second function from the sandbox without creating a new file yet again. 

I am disinclined to merge this since @josef-pkt has registered strong opposition.

**Pull request closed without merging**